software-engineering-policies/InfrastructureAsCode/terraform.md

```diff
@@ -0,0 +1,38 @@
+# Infrastructure as Code: Terraform
+
+Terraform is our tool of choice, and should be considered in the first instance.
+
+Most standards and practises at the UKHO is covered in the Training courses we provide around the use of Terraform.
+
+## Avoid adding local modules
+
+Holding modules in the same repository as the project reduces it's audience. It is likely that you might be tempted to copy the modules you created in the previous project to your new one.
+
+This should be avoided at all costs. Most modules can be written in a relative IP lossless way, to avoid attack vertors or concerns around security, using public modules makes reuse easy to find and use when adhering to a standard naming makes them  easy to find in GitHub, using releases allows for ease of versioning.
```

Comment:
```suggestion
This should be avoided at all costs. Most modules can be written in a relative IP lossless way to avoid attack vectors or concerns around security. Using public modules makes them easy to find and reuse, adhering to a standard naming convention makes them easy to find in GitHub, and using releases allows for ease of versioning.
```

Reply:
applied in other update